test/unit/PanoImageRenderer/WebGLUtils.spec.js

```diff
@@ -43,8 +43,7 @@ describe("WebglUtils", function() {
 
         [
             {os: {name: "android", version: "4.3"}, browser: {name: "chrome"}},
-            {os: {name: "android", version: "4.4"}, browser: {name: "samsung internet"}},
-            {os: {name: "ios", version: "7"}, browser: {name: "safari"}}
```

Comment:
Why remove `{os: {name: "ios", version: "7"}, browser: {name: "safari"}}`?

Reply:
iOS 7 safari don't support webgl. So I removed conditional logic in isStableWebGL method. and added checking webgl support using isWebGLAvailable method before conditional logic in isStableWebGL.

In real iOS7 device, `isStableWebGL` will return `false` because it is not support webgl.
It's not valid test anymore before mocking no-webgl environment.